lms/djangoapps/certificates/signals.py

```diff
@@ -142,3 +144,19 @@ def fire_ungenerated_certificate_task(user, course_key, expected_verification_st
             kwargs['expected_verification_status'] = unicode(expected_verification_status)
         generate_certificate.apply_async(countdown=CERTIFICATE_DELAY_SECONDS, kwargs=kwargs)
         return True
+
+
+@receiver(USER_CERTIFICATE_DOWNLOADABLE)
+def send_email_user_certificate_downloadable(sender, first_name, display_name, certificate_reverse_url, user_email,
+                                             *args, **kwargs):
+    template = MandrillClient.DOWNLOAD_CERTIFICATE
+
+    context = dict(first_name=first_name, course_name=display_name,
+                   certificate_url=certificate_reverse_url)
+    try:
+        MandrillClient().send_mail(template, user_email, context)
+    except Exception as e:
+        # Mandrill errors are thrown as exceptions
+        log.error(
+            'Unable to send email for USER_CERTIFICATE_DOWNLOADABLE signal: {class_name} - {exception_object}'.format(
+                class_name=e.__class__, exception_object=e))
```

Comment:
Please move this code and relevant imports to PhilU files to avoid core changes

Reply:
@zeshanarifios where is that philu file??

Reply:
moved